tox.ini

```diff
@@ -3,6 +3,7 @@
 [tox]
 # Also see .travis.yml where this is maintained separately.
 envlist=py26,py27,py34,py35,docs
+skip_missing_interpreters = True
```

Comment:
I think I'd rather leave this off. It will only fail locally if an interpreter is missing, which is easy to ignore in tox output. My worry about adding this in the tox config is that if Travis ever loses an interpreter than the tests would still pass.